lib/masamune/transform/define_schema.psql.erb

```diff
@@ -20,6 +20,11 @@
 --  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 --  THE SOFTWARE.
 
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+CREATE EXTENSION IF NOT EXISTS hstore;
+
+SET TIMEZONE TO 'UTC';
```

Comment:
these extensions are necessary - noticed that they were missing while testing `thor psql`